CorsixTH/Lua/graphics.lua

```diff
@@ -393,6 +393,8 @@ end
 function Graphics:loadFont(sprite_table, x_sep, y_sep, ...)
   -- Allow (multiple) arguments for loading a sprite table in place of the
   -- sprite_table argument.
+  -- TODO: Native number support for e.g. Korean languages. Current use of cach_x_sep is a stopgap solution for #1193 and should be eventually removed
+  local cache_x_sep = x_sep
```

Comment:
This function is polymorphic, though I don't know it's ever used that way - as such the arguments have the wrong name for what you're using them for, but your variable probably shouldn't. Call it `font_name` or something.